sql/src/main/java/org/apache/druid/sql/calcite/planner/SqlStatementHandler.java

```diff
@@ -41,6 +42,8 @@
   void prepare();
   PrepareResult prepareResult();
   PlannerResult plan() throws ValidationException;
+  String statementKind();
+  SqlNode targetDataSource();
```

Comment:
Should we annotate this with `@Nullable`?

Reply:
Good point, done